### DISCUSSION TITLE
【灌水】今天是第八届挺好萌海选赛day6

### DISCUSSION THREAD

**Discussion by hzjwork** [share121/inter-knot#798] [其他]
![微信图片_20240810165206](https://github.com/user-attachments/assets/746f5473-107b-4231-9133-cc247f5f13a9)
https://tieba.baidu.com/p/9123986189?share=9105&fr=sharewise&see_lz=0&share_from=post&sfc=copy&client_type=2&client_version=12.61.1.0&st=1723279851&is_video=false&unique=ACF07A221044EA45B0C9F8AB924B5606
大家投椎名立希一票谢谢喵

**Comment by UlyssesRen** [comment]
就投soyo就投soyo

**Comment by wushanef** [comment]
投了刃牙一票
